code/modules/overmap/ftl_shunt/core.dm

```diff
@@ -0,0 +1,551 @@
+/obj/machinery/ftl_shunt
+	anchored = 1
+	icon = 'icons/obj/shunt_drive.dmi'
+	var/initial_id_tag = "ftl"
+
+/obj/machinery/ftl_shunt/core
+	name = "superluminal shunt core"
+	desc = "An immensely powerful particle accelerator capable of forming a micro-wormhole and shunting an entire ship through it in a nanosecond."
```

Comment:
I know this is just technobabble but particle accelerator so bland, why not some ridiculous shit like `an n-dimensional non-Baryonic brane deformation system`